src/openrct2/ride/TrackDesign.cpp

```diff
@@ -2103,10 +2096,8 @@ void track_design_draw_preview(TrackDesign* td6, uint8_t* pixels)
     td6->cost = cost;
     td6->track_flags = flags & 7;
 
-    CoordsXYZ centre;
-    centre.x = (gTrackPreviewMin.x + gTrackPreviewMax.x) / 2 + 16;
-    centre.y = (gTrackPreviewMin.y + gTrackPreviewMax.y) / 2 + 16;
-    centre.z = (gTrackPreviewMin.z + gTrackPreviewMax.z) / 2;
+    CoordsXYZ centre = { (gTrackPreviewMin.x + gTrackPreviewMax.x) / 2 + 16, (gTrackPreviewMin.y + gTrackPreviewMax.y) / 2 + 16,
```

Comment:
Use `ToTileCentre`. (Add it if CoordsXYZ does not yet have it.)

Reply:
I mean, the variable is called centre, indeed, but `ToTileCentre` floors the value first and then centers it, it doesn't look like it will be the same.

Reply:
It depends on whether `gTrackPreviewMin` and `gTrackPreviewMax` are pre-floored, I'd say.

Let's keep it like this for now, then.